titus-common/src/main/java/com/netflix/titus/common/model/validator/EntityValidator.java

```diff
@@ -9,5 +11,5 @@
  * @param <T> The type of object this EntityValidator validates.
  */
 public interface EntityValidator<T> {
-    Set<ValidationError> validate(T entity);
+    Flux<Set<ValidationError>> validate(T entity);
```

Comment:
Contract description is missing. The expectation here is that always exactly one value is emitted. If there are no errors, this will be an empty set.

Reply:
Changed type to `Mono`